sway/desktop/output.c

```diff
@@ -429,7 +437,53 @@ static void render_container_simple(struct sway_output *output,
  */
 static void render_container_tabbed(struct sway_output *output,
 		struct sway_container *con) {
-	// TODO
+	struct sway_seat *seat = input_manager_current_seat(input_manager);
+	struct sway_container *focus = seat_get_focus(seat);
+
+	for (int i = 0; i < con->children->length; ++i) {
+		struct sway_container *child = con->children->items[i];
+
+		if (child->type == C_VIEW) {
+			if (child->sway_view->border != B_NONE) {
+				struct border_colors *colors;
+				struct wlr_texture *title_texture;
+				if (focus == child) {
```

Comment:
The code after `if (child->sway_view->border == B_NORMAL) {` can be factored out of this `if`.

We may want a function that chooses `colors` and `title_texture` for a child (since this code is duplicated with [line 408](https://github.com/swaywm/sway/pull/1908/files#diff-28cc674457237875f7921b0d29d34ac2R408)).